src/bucket/BucketManagerImpl.cpp

```diff
@@ -963,7 +967,8 @@ BucketManagerImpl::loadPoolShareTrustLinesByAccountAndAsset(
     AccountID const& accountID, Asset const& asset) const
 {
     releaseAssertOrThrow(getConfig().isUsingBucketListDB());
-    auto timer = getBulkLoadTimer("poolshareTrustlines").TimeScope();
+    // We don't know how many entries this will load, so do not record metric
```

Comment:
I think I need to reword this comment to make it more clear, but the number of entries loaded for the poolshare trustlines and inflation winners is meaningless. For these queries, we need to do a linear search of certain sections of the BucketList, so it doesn't matter how many entries we actually load because we read the entire section anyway. How about I clarify the comment and change the function name to something like "getBulkLoadMetrics"?

Reply:
sounds good. Please squash commits, and we'll merge